styles/blocks/uppercase.json

```diff
@@ -0,0 +1,15 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/theme.json",
+	"version": 3,
+	"slug": "uppercase",
+	"title": "Uppercase",
```

Comment:
I also tried "Bold Uppercase" but it did not fit inside the button in the interface, it got truncated.